recipes/phabox/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "PhaBOX" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: https://github.com/KennthShang/{{ name }}/archive/refs/tags/v{{ version }}-beta.tar.gz
+    sha256: 206d7c5fef0b9cc5fa51e1ed66c77c1f53da47bbdaed1e6f2c0058f8a8aaa403
+    patches:
+      - imports.patch
+  # add setup.py based on https://github.com/KennthShang/PhaBOX/blob/243f78aa82f682c9b8370027c882a4658b9d2945/setup.py
+  # added package_data and entry_points
+  - path: setup.py
```

Comment:
Do this in a build.sh and put comments in there. You can copy the setup.py and then run `"{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv"`